src/constants.js

```diff
@@ -0,0 +1,17 @@
+const COLLECTION_IDS = {
+  Wallpapers: ["137627","1127828","343012","220388","151521","403","402","335992","1494572","1610448"],
```

Comment:
Just find one collection that is the most appropriate for `Wallpapers`. Same goes for the other collections